File: JOSE C/TRABAJOnJSnLPPnPHPnPYTHON___9064b6ef592a541___/TRABAJO-JS-LPP-PHP-PYTHON/pyton/5evidencia.py
def calcular_jornal(turno, horas_trabajadas):
    tarifa_diurna = 10
    tarifa_nocturna = 15

    if turno.lower()== "diurno":
        valor_a_pagar = tarifa_diurna * horas_trabajadas
    elif turno.lower() == "noche":
        valor_a_pagar = tarifa_nocturna * horas_trabajadas
    else:
        valor_a_pagar = 0

    return valor_a_pagar

turno_trabajo = input("Ingrese el turno de trabajo (diurno/noche): ")
horas_trabajadas = int(input("Ingrese el número de horas trabajadas: "))

jornal = calcular_jornal(turno_trabajo, horas_trabajadas)
print("El valor a pagar es de $" , str(jornal))

File: JOSE C/TRABAJOnJSnLPPnPHPnPYTHON___9064b6ef592a541___/TRABAJO-JS-LPP-PHP-PYTHON-7 ejercicios/java/5EVIDENCIA.js
// Solicitar el turno de trabajo y el número de horas trabajadas al usuario
var turno = prompt("Ingrese el turno de trabajo (diurno/nocturna):");
var horasTrabajadas = parseInt(prompt("Ingrese el número de horas trabajadas:"));

var tarifaHora = 0;

if (turno === "diurno") {
  tarifaHora = 10;
} else if (turno === "nocturna") {
  tarifaHora = 15;
} else {
  console.log("Turno no válido. Por favor, ingrese 'diurno' o 'nocturna'.");
}

// Calcular el valor a pagar
var valorPagar = tarifaHora * horasTrabajadas;


console.log("El valor a pagar es: $" + valorPagar);
